.readthedocs.yml

```diff
@@ -16,6 +16,7 @@ build:
     post_create_environment:
       - npm install -g @mermaid-js/mermaid-cli
     pre_build:
+      - pip list
```

Comment:
If we're to have this, let's make it a `pip freeze` instead so we can copy-paste the output to a `requirements.txt`